MagicBytesValidator/Services/Formats/Gif.cs

```diff
@@ -0,0 +1,16 @@
+using MagicBytesValidator.Models;
+
+namespace MagicBytesValidator.Services.Formats
```

Comment:
```suggestion
namespace MagicBytesValidator.Formats
```
Namespace passt nicht so ganz